substrate/frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -393,7 +393,7 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 					#(
 						#cfg_attrs
 						Self::#fn_name { #( #args_name_pattern_ref, )* } => {
-							#feeless_check_result
+							#feeless_check(origin, #( #args_name, )*)
```

Comment:
There is a clippy warning:
```
Run forklift cargo clippy --all-targets --locked --workspace --quiet
   Using forklift, storage: gcs, compressor: zstd-3, uploader threads: 6
error: try not to call a closure in the expression where it is declared
   --> substrate/frame/system/src/lib.rs:649:12
    |
649 |     #[pallet::call]
    |               ^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_closure_call
    = note: `-D clippy::redundant-closure-call` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::redundant_closure_call)]`
```

maybe add `#[allow(clippy::redundant_closure_call)]` or use a variable for the closure: `let c = #feeless_check; c(origin, ...);`